libpod/define/container.go

```diff
@@ -7,6 +7,9 @@ const (
 	RestartPolicyNone = ""
 	// RestartPolicyNo is identical in function to RestartPolicyNone.
 	RestartPolicyNo = "no"
+	// RestartPolicyNever is the same as no, which indicates that the container
+	// should not be restarted when exited.
+	RestartPolicyNever = "never"
```

Comment:
No need for a new const.  Let's just map the string "never" to `RestartPolicyNo` when parsing.

Reply:
fixed